swift/Todo/Model/ObservableQuery.swift

```diff
@@ -32,23 +32,16 @@ class ObservableQuery : ObservableObject {
     
     // Wrapper around Result that conforms to identifiable so we can use it in a SwiftUI List
     struct IResult : Identifiable {
-        public let id: UUID
+        public let id: String
         public let wrappedResult: CouchbaseLiteSwift.Result
-        public var docID: String { // computed docID
-            self.id.uuidString
-        }
         
         init(_ cblResult: CouchbaseLiteSwift.Result) {
             self.wrappedResult = cblResult
             guard let docID = cblResult.string(at: 0)
```

Comment:
To be specific that "id" is required, this should be `docID = cblResult.string(forKey: "id")`.